main.tf

```diff
@@ -6,9 +6,11 @@ module "final_snapshot_label" {
 }
 
 locals {
-  computed_major_engine_version = var.engine == "postgres" ? join(".", slice(split(".", var.engine_version), 0, 1)) : join(".", slice(split(".", var.engine_version), 0, 2))
+  is_postgres                   = var.engine == "postgres" ? true : false
+  computed_major_engine_version = local.is_postgres ? join(".", slice(split(".", var.engine_version), 0, 1)) : join(".", slice(split(".", var.engine_version), 0, 2))
```

Comment:
Instead of creating this `is_postgres` logic in `local.computed_major_engine_version`, wouldn't it be better to create the logic in `local.major_engine_version` instead ?